devito/mpi/halo_scheme.py

```diff
@@ -366,6 +410,10 @@ def distributed_aindices(self):
     def loc_indices(self):
         return set().union(*[i.loc_indices.keys() for i in self.fmapper.values()])
 
+    @cached_property
+    def loc_values(self):
```

Comment:
Is there a better way to get the values of loc_indices?
If so happy to drop this